src/Console/ConsoleIo.php

```diff
@@ -551,8 +550,7 @@ public function createFile($path, $contents, $forceOverwrite = false)
 
         if (file_exists($path) && $forceOverwrite === false) {
             $this->warning("File `{$path}` exists");
-            $key = $this->askChoice('Do you want to overwrite?', ['y', 'n', 'a', 'q'], 'n');
-            $key = strtolower($key);
+            $key = strtolower($this->askChoice('Do you want to overwrite?', ['y', 'n', 'a', 'q'], 'n'));
```

Comment:
Doing more than a single operation per line is usually considered bad practice.
Harder to debug and easier to overlook issues.